crates/torii/sqlite/src/lib.rs

```diff
@@ -173,18 +172,17 @@ impl Sql {
     }
 
     pub async fn cursors(&self) -> Result<Cursors> {
-        let mut conn: PoolConnection<Sqlite> = self.pool.acquire().await?;
         let cursors = sqlx::query_as::<_, (String, String)>(
             "SELECT contract_address, last_pending_block_contract_tx FROM contracts WHERE \
              last_pending_block_contract_tx IS NOT NULL",
         )
-        .fetch_all(&mut *conn)
+        .fetch_all(&self.pool)
         .await?;
 
         let (head, last_pending_block_tx) = sqlx::query_as::<_, (Option<i64>, Option<String>)>(
             "SELECT head, last_pending_block_tx FROM contracts WHERE 1=1",
```

Comment:
Curiosity, why this `1=1` here?

Reply:
Acquire means you need to handle closing the connection yourself etc.. passing the pool directly will manage acquiring and cleaning it

Reply:
> Curiosity, why this `1=1` here?

idk